include/buddy/lwipopts.h

```diff
@@ -9,26 +9,30 @@ extern "C" {
 
 #include <stdint.h>
 
-#define WITH_RTOS            1
-#define MEM_LIBC_MALLOC      0
-#define CHECKSUM_BY_HARDWARE 0
-#define LWIP_DHCP            1
-#define MEM_ALIGNMENT        4
-#define MEMP_NUM_SYS_TIMEOUT 8
-#define LWIP_ETHERNET        1
-#define LWIP_DNS_SECURE      7
-#define DNS_MAX_NAME_LENGTH  128
-
-#define TCP_MSS                1024
-#define TCP_WND                (8 * TCP_MSS)
-#define TCP_SND_BUF            (2 * TCP_MSS)
-#define LWIP_WND_SCALE         0
-#define TCP_RCV_SCALE          0
-#define PBUF_POOL_SIZE         10
-#define PBUF_POOL_SMALL_SIZE   12
-#define IP_REASS_MAX_PBUFS     15
-#define TCPIP_THREAD_STACKSIZE 1248
-#define TCPIP_MBOX_SIZE        PBUF_POOL_SIZE + PBUF_POOL_SMALL_SIZE
+#define WITH_RTOS                    1
+#define MEM_LIBC_MALLOC              0
+#define CHECKSUM_BY_HARDWARE         0
+#define LWIP_DHCP                    1
+#define MEM_ALIGNMENT                4
+#define MEMP_NUM_SYS_TIMEOUT         8
+#define LWIP_ETHERNET                1
+#define LWIP_DNS_SECURE              7
+#define DNS_MAX_NAME_LENGTH          128
+#define DNS_MAX_NAME_LENGTH_EEPROM   60
```

Comment:
Is this the right place for the constant? This thing is mostly for configuring LwIP itself.

Reply:
normally it's DNS_MAX_NAME_LENGTH as already defined in the LWIP stack. I tried to use it in the constants.hpp but that isn't used by the lwip stack on init (and cannot be imported).

Reply:
another option (maybe better) would be to use DNS_MAX_NAME_LENGTH-68 instead.

Reply:
Used now DNS_MAX_NAME_LENGTH-68.